docs/processing_data.rst

```diff
@@ -152,6 +173,235 @@ Although this is a simple example of labeling data other things can be done here
+
++-----------------+
+| |flow_dragging| |
++-----------------+
+
+In this case the filter called *rename* has been dragged from its position between *Sinusoid* and *SineRMS* and will be dropped on the connection coming from *SinRMS* to the storage layer.
+
+Releasing the mouse button to drop the filter on the connection will cause the pipeline to be drawn with the new filter order.
+
++------------------+
+| |flow_reordered| |
++------------------+
+
+Accessing An Existing Pipeline
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To access an existing pipeline from the *South* or *North* screen you may either click on the service name or select the *Details* item from the services context menu.
+
++----------------+
+| |flow_details| |
++----------------+
+
+Canvas Action Bar
+~~~~~~~~~~~~~~~~~
+
+At the top of the working canvas for the pipeline flow editor a small action bar allows for some actions related to the drawing of the pipeline.
+
++------------------+
+| |flow_actionbar| |
++------------------+
+
+From left to right in the action bar the operations support are:
+
+  - **Reload** - reloads the pipeline, discarding any part complete actions.
+
+  - **Reset** - Resets the canvas to include the pipeline, useful if you have scrolled or zoomed the display and want to revert to seeing the entire pipeline.
+
+  - **Undo** - Undo the last action.
+
+  - **Redo** - Redo the last action.
+
+  - **Delete** - Deletes the currently selected item in the pipeline. This may be a filter or connection.
+
+Also displayed in the bottom right of the working canvas is an overview "map" that shows the working area, the pipeline and the area shown currently.  You can click and drag on this to scroll the visible area of the pipeline canvas.
+
+Pipeline Branching
+------------------
+
+It is possible, using the graphical pipeline development view to create pipelines that are not linear. This allows for data to be sent via two parallel branches of a pipeline and processed in parallel. This can be very useful when two sets of operations are required to be performed on the same data and you wish to not have the data from both operations combined in a single assets or pipeline. An example of this might be the processing of vibration data. You may wish to run an FFT filter over the data to examine the frequency distribution of the signals, but also calculate the power in the vibration using the RMS filter. It is unlikely that you want to do these two different approaches to the analysis of the vibration data in the same asset.
+
+To create a branch, drag the filter from the bottom right corner and drop it onto the canvas in free space.
+
++------------+
+| |branch_1| |
++------------+
+
+Not click on the green circle on the output side of the pipeline element from which the branch is to start and drag out a connection.
```

Comment:
"_Not click on the green circle._" I think you mean "_Click on the green circle._"